src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1940,7 +1940,6 @@ because xbuild doesn't support framework reference assemblies.
     <_ProguardConfiguration Include="$(ProguardConfigFiles)" />
   </ItemGroup>
   <ItemGroup Condition=" '$(ProguardConfigFiles)' == '' ">
-    <_ProguardConfiguration Include="$(_AndroidSdkDirectory)tools\proguard\proguard-android.txt" />
```

Comment:
Ok, removing this really broke the `LibraryProjectZipWithLint` test. Is there a real issue there?